src/DevilDaggersInfo.Tools/ContentManager.cs

```diff
@@ -24,12 +24,13 @@ public static void Initialize()
 		if (!Directory.Exists(UserSettings.Model.DevilDaggersInstallationDirectory))
 			throw new InvalidGameInstallationException("Installation directory does not exist.");
 
-		// TODO: Use correct Linux file name for executable.
 #if WINDOWS
 		string ddExe = Path.Combine(UserSettings.Model.DevilDaggersInstallationDirectory, "dd.exe");
+#elif LINUX
+		string ddExe = Path.Combine(UserSettings.Model.DevilDaggersInstallationDirectory, "devildaggers");
+#endif
 		if (!File.Exists(ddExe))
```

Comment:
Perhaps we should include a default case, or not perform this check when WINDOWS and LINUX are both undefined. The build will fail otherwise and we might get a Mac build in the future.